src/components/presentational/InboxResourceDisplay/InboxResourceDisplay.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './InboxResourceDisplay.css'
+import { InboxMessageResource, InboxResource } from '@careevolution/mydatahelps-js'
+import '@fortawesome/fontawesome-svg-core/styles.css';
+import UnstyledButton from '../UnstyledButton/UnstyledButton';
+import Button from '../Button/Button';
+import { noop } from '../../../helpers/functions';
+import language from '../../../helpers/language';
+
+export interface InboxResourceDisplayProps {
```

Comment:
The presentation components don't look like they need to be directly tied to Inbox objects. Is it possible to decouple the inbox aspect? 
Reflecting on studies that have asked for resource lists, this would satisfy showing a resource list that doesn't have to be inbox driven.

Reply:
I think that would be okay for this component.  I'll refactor it to make it reusable outside of the Inbox.

On a similar note to the dimensional vars discussion, I do worry about the possibility of us oversharing non-building-block components in some cases.  The more places we use a given component, the more places have to be okay with that component changing in response to a need elsewhere.  For something complex, it makes sense to move the complex logic into reusable code, but I think it can sometimes make just as much sense to have "your own copy" of less complex code, depending on the context, so you can modify it as needed for a particular use case without affecting other implementations across and outside the library.  I think we just have to try to strike the right balance between reusability/interdependence and context specific customization.

Reply:
Again, interested to hear others thoughts on the above. ^

Reply:
I reworked the `InboxResourceDisplay` component so that it is now a `Resource` component that is not tied to the Inbox.

I would like to keep the "list item" components tied to the Inbox for now though.  They have some very specific inbox list functionality built into them.